tasty-plutus/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ This format is based on [Keep A Changelog](https://keepachangelog.com/en/1.0.0).
 
 ### Added
 
+* Record fields for `ContextBuilder`:
+  * `cbInputs`
+  * `cbOutputs`
+  * `cbSignatories`
+  * `cbDatums`
+  * `cbMinting`
+
```

Comment:
This is a minor bump, so let's make it so.

Reply:
Not sure what you mean - I added it to changelog under unreleased and used aversion number used elsewhere. How _should_ it be done?

Reply:
The changelog needs to have a new section, just below `Unreleased`, with a date stamp and a version number, whose minor version (aka, the second number) has been increased by 1. Then, anything added or modified that's publically-visible should have its `since` annotation match this new version.